processors/common/src/main/kotlin/com/driver733/mapstructfluent/TypeUtils.kt

```diff
@@ -57,3 +57,8 @@ fun TypeName.toKotlinType(): TypeName =
             }
         }
     }
+
+fun Element.toParameterSpec() =
+    ParameterSpec(simpleName.toString(), kotlinType(), emptyList())
+
+fun List<ParameterSpec>.joinToString() = joinToString(prefix = ", ", transform = ParameterSpec::name)
```

Comment:
Function statement should start on the next line (like in the other function statements)